js/app/modules/arrangement/sideBySide.js

```diff
@@ -12,7 +12,7 @@ const Knowledge = imports.app.knowledge;
 const Module = imports.app.interfaces.module;
 const Utils = imports.app.utils;
 
-const SPACING_MAX = 50;
+const SPACING_MAX = 0;
```

Comment:
Yes, it's supposed to disable the spacing between the categories; now, the `CardTitle` modules are each given their own spacing. We don't need that responsive spacing anymore because of the new work you did on the dropdown menu; it's more important for the highlighted state of `CardTitle` to look more seamless